api/apps/api/test/project-scenarios.e2e-spec.ts

```diff
@@ -23,40 +26,72 @@ let fixtures: FixtureType<typeof getFixtures>;
 
 beforeEach(async () => {
   fixtures = await getFixtures();
-}, 12_000);
+}, 20000000);
```

Comment:
I assume these are typos - 20M milliseconds is ~5.5hrs 🤔

Reply:
Yes and no. I accidentally left them in, but because I need them locally, since the tests will always time out on my machine hahaha